config/locales/en/models/claim.yml

```diff
@@ -262,6 +284,25 @@ en:
               too_long_before_earliest_reporder: The graduated fee date cannot be no earlier than two years before the earliest representation order date
             quantity:
               numericality: Enter a valid quantity for the graduated fee
+        fee/interim_fee:
+          attributes:
+            amount:
+              blank: Enter an amount for the interim fee
+              item_max_amount: The amount for the interim fee exceeds the limit
+              numericality: Enter a valid amount for this fee
+              present: Do not enter an amount for the interim fee
+            fee_type_id:
+              blank: Choose a type for the interim fee
+            quantity:
+              blank: Enter a PPE quantity for the interim fee
+              numericality: Enter a valid PPE quantity for the interim fee
```

Comment:
Does this `quantity` needs `item_max_amount` key?

Reply:
`item_max_amount` key added